packages/dashboard/src/app/routes/_authenticated/_payment-methods/payment-methods_.$id.tsx

```
@@ -39,7 +39,7 @@ export const Route = createFileRoute('/_authenticated/_payment-methods/payment-m
         queryDocument: paymentMethodDetailDocument,
         breadcrumb(_isNew, entity) {
             return [
-                { path: '/payment-methods', label: 'Payment methods' },
+                { path: '/payment-methods', label: 'Payment Methods' },
                 _isNew ? <Trans>New payment method</Trans> : entity?.name,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)